Cinema system/Cinema system/Models/Cinema.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Cinema_system.Models
```

Comment:
Use CamelCase in namespaces. Also preferably stick to using [Organization].[Product].[Namespace].[SubNameSpace].[SubSubNamespace].[...] pattern